tools/functional-tester/test

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+set -x
+set -e
+
+# make sure it's linux64 
+export GOOS=linux
+export GOARCH=amd64
+
+# 1. build etcd binaries
+[ -f bin/etcd ] || ./build
```

Comment:
If someone on OS X runs `./build`, then runs this script, `bin/etcd` won't be of the right os, right?